docs/manual.rst

```diff
@@ -2409,37 +2409,62 @@ STR-packed R-tree
 
 Shapely provides an interface to the query-only GEOS R-tree packed using the
 Sort-Tile-Recursive algorithm. Pass a list of geometry objects to the STRtree
-constructor to create an R-tree that you can query with another geometric object.
+constructor to create a spatial index that you can query with another geometric
+object. Query-only means that once created, the `STRtree` is immutable. You
+cannot add or remove geometries.
 
 .. class:: strtree.STRtree(geometries)
 
   The `STRtree` constructor takes a sequence of geometric objects.
 
-  These are copied and stored in the R-tree.
+  These are copied and stored in the R-tree. However, references are kept to
+  the original geometries and queries return these, not copies.
```

Comment:
Let's delete the original text, which is in fact erroneous, and say something like "references to these geometric objects are stored in the R-tree".

Reply:
af89f54000e95ad99095e64b92082fc751fd4aab